schemas.md

```diff
@@ -43,6 +43,19 @@ The user's geolocation at that time.
 - `description`: Text description, such as messages sent by a GPS beacon.
 - `mixin:location`
 
+### activity.motion
```

Comment:
I don't have Google Fit, but does the data include geolocation data?

Reply:
You've lead me to discover that there is a setting in Fit to enable geolocation. Currently I do not have any geolocation, but I think there is. I will come back after some exercise to examine the new data type

Reply:
Though there is geolocation gather from Fit, the data source this code examines does not show the geolocation, so there is no geolocation schema needed